### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by itdove**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - [Pre-check before submitting a PR](#pre-check-before-submitting-a-pr)
+    - [Build and push images](#build-and-push-images)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Contributing guidelines
+
+## Sign the CLA
+
+IBM cloud projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests.  Please see xxx for more info
```
Comment (qiujian16) [inline CONTRIBUTING.md:18]: we will avoid mention CLA, RH uses DCO more. here is an example https://github.com/open-cluster-management/api/pull/65

File: pkg/collectors/managedclusterinfo.go [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+			Help: descClusterInfoHelp,
+			GenerateFunc: wrapManagedClusterInfoFunc(func(mciObj *unstructured.Unstructured) metric.Family {
+				mci := &mciv1beta1.ManagedClusterInfo{}
+				err := runtime.DefaultUnstructuredConverter.FromUnstructured(mciObj.UnstructuredContent(), &mci)
+				if err != nil {
+					return metric.Family{Metrics: []*metric.Metric{}}
+				}
+				if mci.Status.ClusterID == "" ||
+					mci.Status.KubeVendor == "" ||
+					mci.Status.CloudVendor == "" ||
```
Comment (qiujian16) [inline pkg/collectors/managedclusterinfo.go:77]: we do not always return cloud vendor, what if it is baremetal?

File: pkg/collectors/managedclusterinfo.go [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+		Resource: "managedclusterinfos",
+	}
+)
+
+func getHubClusterId(c dynamic.Interface) string {
+
+	cvObj, errCv := c.Resource(cvGVR).Get(context.TODO(), "version", metav1.GetOptions{})
+	if errCv != nil {
+		klog.Fatalf("Error getting cluster version %v \n", errCv)
+		panic(errCv.Error())
```
Comment (qiujian16) [inline pkg/collectors/managedclusterinfo.go:51]: I think you would not need panic if fatal in advance.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,85 @@
+language: go
```
[the remark this answers is not in the document]
Comment (itdove) [reply .travis.yml:1]: We considered to use prow but we have no time to do it in this release.

File: CONTRIBUTING.md (hunk 1 above)
Comment (itdove) [reply CONTRIBUTING.md:18]: good point... thx.

File: pkg/collectors/managedclusterinfo.go (hunk 2 above)
Comment (itdove) [reply pkg/collectors/managedclusterinfo.go:77]: The CloudVendor is empty if baremetal or shows Other "CloudVendorOther"? 
What attribute shows that is baremetal?

File: pkg/collectors/managedclusterinfo.go (hunk 3 above)
Comment (itdove) [reply pkg/collectors/managedclusterinfo.go:51]: ???
this is evaluated at startup... so if panic the pod will restart until fixed.

Comment (qiujian16) [reply pkg/collectors/managedclusterinfo.go:51]: I think klog.Fatal already makes it exit? https://godoc.org/k8s.io/klog#Fatal

File: pkg/collectors/managedclusterinfo.go (hunk 2 above)
Comment (qiujian16) [reply pkg/collectors/managedclusterinfo.go:77]: oh yeah, you are right. It return "Other" https://github.com/open-cluster-management/multicloud-operators-foundation/blob/60ef45157161b6d4ec2af817fb95d8e689a480e7/pkg/klusterlet/clusterinfo/clusterinfo_controller.go#L354

File: pkg/collectors/managedclusterinfo.go (hunk 3 above)
Comment (itdove) [reply pkg/collectors/managedclusterinfo.go:51]: oh... of course

**Review by leena-jawale — APPROVED** [review]

**Pull request merged**